## What is PRIDE Archive

The PRIDE PRoteomics IDEntifications (PRIDE) Archive database is a centralized, standards compliant, public data repository for mass spectrometry proteomics data, including protein and peptide identifications and the corresponding expression values, post-translational modifications and supporting mass spectra evidence (both as raw data and peak list files). PRIDE is a core member in the ProteomeXchange (PX) consortium, which provides a standardised way for submitting mass spectrometry based proteomics data to public-domain repositories. Datasets are submitted to ProteomeXchange via PRIDE and are handled by expert bio-curators. All PRIDE public datasets can also be searched in ProteomeCentral, the portal for all ProteomeXchange datasets.

All public datasets available in PRIDE Archive in the repository can be accessed via the:

Website, for users to explore through a web browser. [https://www.ebi.ac.uk/pride/archive/](https://www.ebi.ac.uk/pride/archive/)

Web Service, for programmatic access, query and use PRIDE Restful API [https://www.ebi.ac.uk/pride/ws/archive/v2/swagger-ui.html](https://www.ebi.ac.uk/pride/ws/archive/v2/swagger-ui.html)

If you have further queries please consult our Help FAQ or please contact us at [pride-support@ebi.ac.uk](pride-support@ebi.ac.uk) for advice.

## PRIDE's mission

- To archive all types of proteomics mass spectrometry data for the purpose of supporting reproducible research, allowing the application of quality control metrics and enabling the reuse of these data by other researchers.

- To integrate MS-based data in a protein-centric manner to provide information on protein variants, modifications, and expression.

- To provide mass spectrometry based expression data to the Expression Atlas. 

## PRIDE data policy 

PRIDE data policy can be read [here](./datapolicy) 

## How to cite PRIDE Resources

All PRIDE Archive and PX identifiers for a dataset.

## Citing PRIDE Resources

- **Citing the PRIDE database generally**: Perez-Riverol Y, Bai J, Bandla C, García-Seisdedos D, Hewapathirana S, Kamatchinathan S, Kundu DJ, Prakash A, Frericks-Zipper A, Eisenacher M, Walzer M, Wang S, Brazma A, Vizcaíno JA. [The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences](https://academic.oup.com/nar/article/50/D1/D543/6415112). Nucleic Acids Res. 2022 Jan 7;50(D1):D543-D552. doi: 10.1093/nar/gkab1038. PMID: 34723319; PMCID: PMC8728295.

- **Citing ProteomeXchange**: Deutsch EW, Csordas A, Sun Z, Jarnuczak A, Perez-Riverol Y, Ternent T, Campbell D, Bernal-Llinares M, Okuda S, Kawano S, Moritz RL,1 Carver JJ., Wang M, Ishihama Y, Bandeira N, Hermjakob H and Vizcaíno JA.
  [The ProteomeXchange consortium in 2017: supporting the cultural change in proteomics public data deposition](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210636/).  Nucleic Acids Res. 2017 Jan 4; 45(Database issue): D1100–D1106. PubMed ID: 27924013.

- **When a dataset has been deposited in PRIDE (as part of ProteomeXchange) the corresponding manuscript should contain the following statement**:

  **PX Partial**: The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository [1] with the dataset identifier PXDxxxxxx.

  **PX Complete**: The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium (http://proteomecentral.proteomexchange.org) via the PRIDE partner repository [1] with the dataset identifier PXDxxxxxx and DOI 10.6019/PXDxxxxxx.

  [1] Perez-Riverol Y, Bai J, Bandla C, García-Seisdedos D, Hewapathirana S, Kamatchinathan S, Kundu DJ, Prakash A, Frericks-Zipper A, Eisenacher M, Walzer M, Wang S, Brazma A, Vizcaíno JA. [The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences](https://academic.oup.com/nar/article/50/D1/D543/6415112). Nucleic Acids Res. 2022 Jan 7;50(D1):D543-D552. doi: 10.1093/nar/gkab1038. PMID: 34723319; PMCID: PMC8728295.

  See some examples: [PX Partial Submission](https://www.sciencedirect.com/science/article/pii/S0092867416315914), [PX Complete Submission](https://www.frontiersin.org/articles/10.3389/fmicb.2018.00680/full).

## Citing PRIDE Peptidome/PRIDE Cluster

- **Citing PRIDE Cluster**: Griss J, Perez-Riverol Y, Lewis S, Tabb DL, Dianes JA, Del-Toro N, Rurik M, Walzer MW, Kohlbacher O, Hermjakob H, Wang R, Vizcaíno JA. [Recognizing millions of consistently unidentified spectra across hundreds of shotgun proteomics datasets](https://www.ncbi.nlm.nih.gov/pubmed/27493588). Nat Methods. 2016 Aug;13(8):651-656. PubMed ID: 27493588.

## Citing PRIDE Tools

- **PRIDE Inspector Toolsuite**: Perez-Riverol Y, Xu QW, Wang R, Uszkoreit J, Griss J, Sanchez A, Reisinger F, Csordas A, Ternent T, del Toro N, Dianes JA, Eisenacher M, Hermjakob H, Vizcaíno JA. [PRIDE Inspector Toolsuite: moving towards a universal visualization tool for proteomics data standard formats and quality assessment of ProteomeXchange datasets.](http://www.mcponline.org/content/15/1/305.long). Mol Cell Proteomics 2016 Jan; 15(1):305-17. PubMed ID: 26545397.

- **PRIDE Cluster** or **Spectra Cluster Toolsuite**: Griss J, Perez-Riverol Y, Lewis S, Tabb DL, Dianes JA, Del-Toro N, Rurik M, Walzer MW, Kohlbacher O, Hermjakob H, Wang R, Vizcaíno JA. [Recognizing millions of consistently unidentified spectra across hundreds of shotgun proteomics datasets](https://www.ncbi.nlm.nih.gov/pubmed/27493588). Nat Methods. 2016 Aug;13(8):651-656. PubMed ID: 27493588.


All manuscripts from the PRIDE team can be found in [PRIDE Team GoogleScholar](https://scholar.google.co.uk/citations?user=tzHv1ZcAAAAJ&hl=en)

## The team behind PRIDE

The [PRIDE team](https://www.ebi.ac.uk/about/teams/proteomics/members/) currently led by [Juan Antonio Vizcaino](https://www.ebi.ac.uk/about/people/juan-vizcaino), consists of software developers, biocurators, trainees/visitors and one team coordinator.

## The members of the PRIDE Scientific Advisory Board

The PRIDE project receives advice and guidance from our scientific advisory board. The PRIDE SAB consists of a collection of scientists representing different groups working in the mass spectrometry proteomics field. Our current SAB members are:

- Ruedi Aebersold, Swiss Federal Institute of Technology, ETH, Zurich, Switzerland
- Jurgen Cox, Max Planck Institute of Biochemistry, Martinsried, Germany
- Pedro R. Cutillas, Queen Mary University London, UK
- Concha Gil , Universidad Complutense Madrid, Spain
- Juri Rappsilber, University of Edinburgh, UK
- Hans Vissers, Waters Corporation, Wilmslow, UK

We also would like to thank all previous members of the PRIDE SAB:

- Roz Banks, University of Leeds, UK
- Angus Lamond, University of Dundee, UK
- Kathryn Lilley, University of Cambridge, UK
- Ioannis Xenarios, SIB Swiss Institute of Bioinformatics, Geneva, University of Lausanne, Switzerland  

## Acknowledgements

The PRIDE team would like to thank all data submitters for their contributions. The PRIDE team is funded at present by the Wellcome Trust [grant number 208391/Z/17/Z],  the BBSRC grants BB/N022440/1, BB/N022432/1 and BB/P024599/1, the NIH grant R24 GM127667-01, the H2020 EU EPIC-XS grant [grant number 823839], several ELIXIR Implementation Studies and EMBL core funding. 

In recent years, PRIDE has also been funded by the Wellcome Trust [grant numbers WT101477MA and WT085949MA], the EU FP7 grants 'ProteomeXchange’ [grant number 260558], ‘PRIME-XS’ [grant number 262067], and ‘LipidomicNet’ [grant number 202272], and the BBSRC grants [reference BB/I024204/1, BB/I00095X/1, BB/K01997X/1 and BB/L024225/1].


## Clinical (potentially sensitive) clinical human datasets

All datasets in PRIDE (as part of ProteomeXchange) are made fully open, once the corresponding paper is published. As such, accessed-controlled datasets are not supported at present. If you have concerns about submitting datasets to PRIDE due to legal issues, please contact [pride-support](pride-support@ebi.ac.uk). 